net.openra.OpenRA.json

```diff
@@ -0,0 +1,328 @@
+        "/bin/mono-service",
+        "/bin/mono-package-runtime",
+        "/bin/mono-heapviz",
+        "/bin/mono-gdb.py",
+        "/bin/mono-find-requires",
+        "/bin/mono-find-provides",
+        "/bin/mono-configuration-crypto",
+        "/bin/mono-cil-strip",
+        "/bin/mono-boehm",
+        "/bin/mono-api-info",
+        "/bin/mono-api-html",
+        "/bin/mod",
+        "/bin/mkbundle",
+        "/bin/mdvalidater",
+        "/bin/mdoc-validate",
+        "/bin/mdoc-update",
+        "/bin/mdoc-export-msxdoc",
+        "/bin/mdoc-export-html",
+        "/bin/mdoc-assemble",
+        "/bin/mdoc",
+        "/bin/mdbrebase",
+        "/bin/mdassembler",
+        "/bin/mcs",
+        "/bin/mconfig",
+        "/bin/makecert",
+        "/bin/macpack",
+        "/bin/lc",
+        "/bin/installvst",
+        "/bin/ilasm",
+        "/bin/ikdasm",
+        "/bin/httpcfg",
+        "/bin/genxs",
+        "/bin/gacutil2",
+        "/bin/gacutil",
+        "/bin/dtd2xsd",
+        "/bin/dtd2rng",
+        "/bin/dmcs",
+        "/bin/disco",
+        "/bin/csi",
+        "/bin/csharp",
+        "/bin/csc",
+        "/bin/crlupdate",
+        "/bin/chktrust",
+        "/bin/certmgr",
+        "/bin/cert2spc",
+        "/bin/cert-sync",
+        "/bin/ccrewrite",
+        "/bin/cccheck",
+        "/bin/caspol",
+        "/bin/al2",
+        "/bin/al"
+      ],
+      "config-opts": [
+        "--enable-minimal=aot,profiler",
+        "--with-mcs-docs=no",
+        "--disable-dependency-tracking"
+      ]
+    },
+    {
+      "name": "libgdiplus",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.mono-project.com/sources/libgdiplus/libgdiplus0-4.2.tar.gz",
+          "sha256": "f332b9b8b44fd1c50b8d8d01a7296360b806c790b8297614739b3de1edbadfeb"
+        }
+      ],
+      "cleanup": [ "/include", "/lib/*.a", "/lib/*.la", "/lib/pkgconfig", "/share/man" ]
+    },
+    {
+      "name": "lua51",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://www.lua.org/ftp/lua-5.1.5.tar.gz",
+          "sha256": "2640fc56a795f29d28ef15e13c34a47e223960b0240e8cb0a82d9b0738695333"
+        },
+        {
+          "type": "patch",
+          "path": "lua51-cflags.patch"
+        },
+        {
+          "type": "patch",
+          "path": "lua51-prefix.patch"
+        },
+        {
+          "type": "patch",
+          "path": "lua51-so.patch"
+        },
+        {
+          "type": "shell",
+          "commands": [
+            "sed -e 's:llua:llua5.1:' -e 's:/include:/include/lua5.1:' -i etc/lua.pc",
+            "sed -r -e '/^LUA_(SO|A|T)=/ s/lua/lua5.1/' -e '/^LUAC_T=/ s/luac/luac5.1/' -i src/Makefile",
+            "mv doc/lua.1 doc/lua5.1.1",
+            "mv doc/luac.1 doc/luac5.1.1"
+          ]
+        }
+      ],
+      "cleanup": [ "/bin", "/include", "/lib/*.a", "/lib/*.la", "/lib/pkgconfig", "/share/man" ],
+      "buildsystem": "simple",
```

Comment:
Any idea if it works with luajit?

Reply:
OpenRA uses **MemoryConstrainedLuaRuntime** from [Eluant](https://github.com/cdhowie/Eluant). Unfortunately is uses a custom memory allocator, which doesn't meet the special needs of LuaJIT on 64 bit.